### PR TITLE
update project for swift 5 :

### PR DESCRIPTION
**Pull request by tirrorex**
This branch includes fixes that prevent the library from being updating to swift 5 (most of those build issues appear only during a pod lib lint though).

Feel

### CODE REVIEW

**Comment by tirrorex** [comment]
@RomanPodymov i am waiting for alamofire to be out of beta, won't be doing any changes until then ;)

File: AlamofireObjectMapper.podspec [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
   s.name = "AlamofireObjectMapper"
-  s.version = "6.0.0"
+  s.version = "6.0.1"
   s.license = { :type => "MIT", :file => "LICENSE" }
   s.summary = "An extension to Alamofire which automatically converts JSON response data into swift objects using ObjectMapper"
-  s.homepage = "https://github.com/tristanhimmelman/AlamofireObjectMapper"
+  s.homepage = "https://github.com/tirrorex/AlamofireObjectMapper"
   s.author = { "Tristan Himmelman" => "tristanhimmelman@gmail.com" }
-  s.source = { :git => 'https://github.com/tristanhimmelman/AlamofireObjectMapper.git', :tag => s.version.to_s }
+  s.source = { :git => 'https://github.com/tirrorex/AlamofireObjectMapper.git', :tag => s.version.to_s }
```
Comment (RomanPodymov) [inline AlamofireObjectMapper.podspec]: Why did you change this URL?

File: AlamofireObjectMapper.podspec [hunk 2]
```diff
@@ -1,22 +1,22 @@
 Pod::Spec.new do |s|
 
   s.name = "AlamofireObjectMapper"
-  s.version = "6.0.0"
+  s.version = "6.0.1"
   s.license = { :type => "MIT", :file => "LICENSE" }
   s.summary = "An extension to Alamofire which automatically converts JSON response data into swift objects using ObjectMapper"
-  s.homepage = "https://github.com/tristanhimmelman/AlamofireObjectMapper"
+  s.homepage = "https://github.com/tirrorex/AlamofireObjectMapper"
```
Comment (RomanPodymov) [inline AlamofireObjectMapper.podspec]: Why did you change this URL?

File: AlamofireObjectMapper.podspec (hunk 1 above)
Comment (tirrorex) [reply AlamofireObjectMapper.podspec]: Forgot about that before doing the pr.
I obviously changed to url to be able to use my code as the source in my project.

File: AlamofireObjectMapper.podspec (hunk 2 above)
Comment (tirrorex) [reply AlamofireObjectMapper.podspec]: Same answer

**Comment by tirrorex** [comment]
Killing the pr for now, don't have time to do the fixes required
